olive/common/config_utils.py

```diff
@@ -159,6 +159,7 @@ class ConfigParam(ConfigBase):
     required: bool = False
     default_value: Any = None
     is_object: bool = False
+    is_path: bool = False
```

Comment:
>is_path: bool = False [](http://example.com/codeflow?start=4&length=21)

does the movement impact the generated document? I see the subclass has document but the base class doesn't have.

Reply:
<img width="788" alt="image" src="https://github.com/microsoft/Olive/assets/13343117/6f95edf9-a833-4967-9bd7-e4c8d7aac398">
I think it is fine. The description all set in `PassConfigParam`